src/txn/txn_rollback_to_stable.c

```diff
@@ -1195,18 +1195,14 @@ __rollback_to_stable_check(WT_SESSION_IMPL *session)
     WT_DECL_RET;
     bool txn_active;
 
-    /*
-     * Help the user comply with the requirement that there are no concurrent operations. Protect
-     * against spurious conflicts with the sweep server: we exclude it from running concurrent with
-     * rolling back the history store contents.
-     */
-    ret = __wt_txn_activity_check(session, &txn_active);
+    /* Help the user comply with the requirement that there are no concurrent user operations. */
+    txn_active = __wt_txn_user_active(session);
```

Comment:
As we are changing the approach of transaction check logic, do we still need the loop logic to wait for the eviction threads?

Reply:
No we don't need to wait for eviction threads to finish off. I removed that code in 871f5b7.